hyperbee/import.mjs

```diff
@@ -153,8 +153,20 @@ for (var key in metaContent) {
         // then store it as an array
         const parentEncryptedArchiveCid = metaContent[key][extrasKey];
         const parentContentCid = cidMapping[parentEncryptedArchiveCid];
-        dbAppend(datadb, contentCID, "childOf", CID.parse(parentContentCid));
-        dbAppend(datadb, parentContentCid, "parentOf", CID.parse(contentCID));
+        await dbAddRelation(
+          datadb,
+          contentCID,
+          "parents",
+          "verified",
+          CID.parse(parentContentCid)
+        );
+        await dbAddRelation(
+          datadb,
+          parentContentCid,
+          "children",
+          "verified",
+          CID.parse(contentCID)
+        );
       }
       dbPut(datadb, contentCID, extrasKey, metaContent[key][extrasKey]);
```

Comment:
I see how it's a bad practice. In this case I wasn't worried about it because the script won't end until all the async functions finish. But if you'd like to stay consistent on this I can change it.

Reply:
yeah, the process won't exit if there's anything on the event loop, that's true. It will exit with an unresolved promise though. Handling the promises explicitly is best practice, but I'll leave it up to you whether changing this file is a good use of time. either way is ok with me.